hardware/openwebnet/bt_openwebnet.h

```diff
@@ -249,6 +270,22 @@ typedef enum {
 	ENERGY_MANAGEMENT_DIMENSION_STATUS_STOPGO_MONTHLY_TOTALIZERS_DAILY_BASIS_LAST_YEAR_COMPARED_TO_32BIT_GRAPHICS_TOUCHX_PREVIUS_YEAR = 514
 }eEnergyManagementDimension;
 
+typedef enum {
```

Comment:
Please remove the typedef as it is not defined.
Could you place this enum in the .cpp file as it is not used in the header (saves compile time)

Reply:
Another developer had declared all the enumerations of this file as typedef with its associated name, even if they are never declared.
Do I have to remove all typedefs and keep only the enums?

The enums should remain in the .h because they could also be used in the OpenWebNetUSB.cpp file when someone will develop new features already present in OpenWebNetTCP.cpp.

Reply:
Yes please remove the keyword 'typedef'